main.py

```diff
@@ -21,10 +21,30 @@ def main():
         SetAPIKey()
     co = DBConnect()
     GetGamesList(co)
-
+    CleanNotInstalledGames(co)
 
 ####### FUNCTIONS
 
+#Get list of installed games
+def CleanNotInstalledGames(co):
```

Comment:
This function first search all installed games and then execute the function DeleteImages() for remove all jpg files for uninstalled games